src/ert/parsing/config_schema_deprecations.py

```diff
@@ -0,0 +1,149 @@
+from .deprecation_info import DeprecationInfo
+
+REPLACE_WITH_GEN_KW = [
+    "RELPERM",
+    "MULTZ",
+    "EQUIL",
+    "GEN_PARAM",
+    "MULTFLT",
+]
+JUST_REMOVE_KEYWORDS = [
+    "UMASK",
+    "LOG_FILE",
+    "LOG_LEVEL",
+    "ENKF_RERUN",
+]
+RSH_KEYWORDS = ["RSH_HOST", "RSH_COMMAND", "MAX_RUNNING_RSH"]
+USE_QUEUE_OPTION = [
+    "LSF_SERVER",
+    "LSF_QUEUE",
+    "MAX_RUNNING_LSF",
+    "MAX_RUNNING_LOCAL",
+]
+
+deprecated_keywords_list = [
+    *[
+        DeprecationInfo(
+            keyword=kw,
+            message=f"Using {kw} with substitution strings"
+            f"that are not of the form '<KEY>' is deprecated."
+            f"Please change to <{kw.replace('<', '').replace('>','')}>",
+            check=lambda line: not DeprecationInfo.is_angle_bracketed(line[0]),
+        )
+        for kw in ["DEFINE", "DATA_KW"]
+    ],
+    DeprecationInfo(
+        keyword="RUNPATH",
+        message=lambda line: "RUNPATH keyword contains deprecated value "
+        f"placeholders: %d, instead use: "
+        f"{line[0].replace('%d', '<IENS>', 1).replace('%d', 'ITER', 1)}",
+        check=lambda line: "%d" in " ".join(line),
+    ),
+    *[
+        DeprecationInfo(
+            keyword=kw,
+            message=f"The {kw} keyword was replaced by the GEN_KW keyword. "
+            "Please see https://ert.readthedocs.io/en/latest/"
+            "reference/configuration/keywords.html#gen-kw "
+            "to see how to migrate from MULTFLT to GEN_KW.",
+        )
+        for kw in REPLACE_WITH_GEN_KW
+    ],
+    *[
+        DeprecationInfo(
+            keyword=kw,
+            message=f"The keyword {kw} no longer has any effect"
+            f", and can be safely removed.",
+        )
+        for kw in JUST_REMOVE_KEYWORDS
+    ],
+    *[
+        DeprecationInfo(
+            keyword=kw,
+            message=f"The {kw} was used for the deprecated and removed "
+            "support for RSH queues. It no longer has any effect "
+            "and can safely be removed from the config file.",
+        )
+        for kw in RSH_KEYWORDS
+    ],
+    *[
+        DeprecationInfo(
+            keyword=kw,
+            message=f"The {kw} keyword has been removed. For most cases this option "
+            "should be set in the site config, and as a regular user you can "
+            "simply remove this from your config. If you need to set these "
+            "options it is now done via the QUEUE_OPTION keyword.",
+        )
+        for kw in USE_QUEUE_OPTION
+    ],
+    DeprecationInfo(
+        keyword="SCHEDULE_PREDICTION_FILE",
+        message="The 'SCHEDULE_PREDICTION_FILE' config keyword "
+        "has been removed and no longer has any effect",
+    ),
```

Comment:
Is this a duplicate? See at the bottom of this list

Reply:
Yep, removing it